src/YAMLValidator/YAMLValidator.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+      <NoWarn>1591, CA1822</NoWarn>
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+      <NoWarn>1591, CA1822</NoWarn>
+    </PropertyGroup>
```

Comment:
Any `<NoWarn>` in the project file should typically have a comment explaining what warnings are being suppressed.  
They're fairly simple to look up, but it's simpler for readers (either in PR or looking over the file in the future) to see it explicitly named.

Reply:
I added an explanation as to why CA1822 was added, the other error was previously present from the Directory.Build.Props